src/views/Layout.jsx

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable jsx-a11y/anchor-is-valid */
-import { Outlet } from 'react-router-dom';
+import { Outlet, NavLink } from 'react-router-dom';
 import { useAuth, SignInButton, SignOutButton } from '../api/useAuth';
-import { NavLink } from 'react-router-dom';
 // import { Home, List, ManageList } from '../views';
 
 import './Layout.css';
```

Comment:
This comment is related to the TODO  comment starting on line 8:

Since we've addressed this TODO as well by leveraging the `NavLink` component, we can get rid of this comment as well! So excited for all this code cleanup!